examples/group/retail_group.json

```diff
@@ -32,9 +32,8 @@
     ]
    },
    "transform": {
-    "build": { 
```

Comment:
I initially wrote the example with build pipeline both because it is very practical and for a more didactical reason. They did not work on Hub -> I went back to a more regular image.